app/controllers/movies_controller.rb

```diff
@@ -1,7 +1,12 @@
 class MoviesController < ApplicationController
 
     def index
-      @movies = Movie.page(params[:page]).per(12)
+      if params[:category] == "Money"
+        movies = Movie.where(category: "Money")
+        @movies = movies.page(params[:page]).per(12).all.order(id: "ASC")
```

Comment:
`else` の場合も含め，`order`を先に書いた方がよいですよ。

```suggestion
        movies = Movie.where(category: "Money").order(id: "ASC")
        @movies = movies.page(params[:page]).per(12)
```